src/Resources/config/block.xml

```diff
@@ -35,7 +35,7 @@
             <tag name="sonata.breadcrumb"/>
             <argument>page</argument>
             <argument>sonata.page.block.breadcrumb</argument>
-            <argument type="service" id="sonata.templating"/>
```

Comment:
hmmm why are you removing `sonata.templating`, we don't need this Sonata Templating anymore?